advanced/Scripts/utils.sh

```diff
@@ -33,3 +33,30 @@ addOrEditKeyValPair() {
     echo "${key}=${value}" >> "${file}"
   fi
 }
+
+#######################
+# returns FTL's current telnet API port
+#######################
+getFTLAPIPort(){
+  local -r FTLCONFFILE="/etc/pihole/pihole-FTL.conf"
+  local -r DEFAULT_PORT_FILE="/run/pihole-FTL.port"
+  local -r DEFAULT_FTL_PORT=4711
+  local PORTFILE
+  local ftl_api_port
+
+  if [[ -f "$FTLCONFFILE" ]]; then
+    # if PORTFILE is not set in pihole-FTL.conf, use the default path
+    PORTFILE="$( (grep "^PORTFILE=" $FTLCONFFILE || echo "$DEFAULT_PORT_FILE") | cut -d"=" -f2-)"
```

Comment:
Can we add a generic `readKey` (or named similarly) function so that we can re-use it? I'm also looking to add a function to remove a key from a file:

https://github.com/pi-hole/pi-hole/commit/9aefc6e9d881171668b12132744553eea3aa859d

Reply:
I think we could do that, but I would prefer to open a new PR for that. One thing - one PR ;-)

Reply:
I would say it's relevant to this PR, personally, but it's OK, I'll add it to the other branch I'm working on once this is merged.

Can you squash/rebase and FP the commits in this PR though please?

Reply:
Done.